### ISSUE TITLE
Titlebars?

### ISSUE THREAD

**Issue by ghost** [vaxerski/Hypr#49]
**Is your feature request related to a problem? Please describe.**
Nope, i just think it would be really cool to have title bars

**Describe the solution you'd like**
Well title bars obviously, maybe something like this? here:
```
Titlebars {
        enabled=1
        size=40
        side=left
        color=0x00000000

        button=left,✕,0xffffffff,killactive,
        button=left,⏹,0xffffffff,fullscreen,
        button=left,☁,0xffffffff,togglefloating,
        button=right,%WINNAME%,0xffffffff,exec,/home/user/somescript.sh
}
```
`button` would be similar to bind in this way:
```
button=SIDE, ICON, COLOR, DISPATCHER, COMMAND
```
and would have additional tokens something along the lines of this:
```
SIDE (left, right, or center.)
ICON (some Unicode text, maybe even an image path perhaps?)
```

I'm new to filing issues, if i have done something incorrectly please inform me! :)

**Comment by vaxerski** [comment]
titlebars aren't really something I would consider useful or needed for a tiling window manager. Is there really any usefulness to them?

The window title, first of all, is pretty useless, second, you can get it in the status bar.

The controls are just slow and you're better off with a keybind.

**Comment by vaxerski** [comment]
a lot of work for a pretty pointless feature. I might add them off in the future, but for now I'd prefer to focus on more important features, bugs and Hyprland.

**Comment by i-pwl** [comment]
fair enough, i had just seen the window animation eye-candy, and thought it would be cool. thanks!
